models/oneinch/fantom/oneinch_fantom_schema.yml

```diff
@@ -0,0 +1,47 @@
+version: 2
+
+models:
+  - name: oneinch_fantom_calls_transfers
```

Comment:
you can add tests in this schema file on the unique keys, to ensure no duplicates. there are lots of examples in other schema files in the repo. it's the "unique combo of columns" test, where you supply the unique keys specified in the model

Reply:
i was thinking to add uniq columns test, but seems that it might be a bit heavy for compute. i'll add after your approve

Reply:
it's good to see that test pass in the CI phase, since we don't run the test in prod. it'll help ensure no duplicates in historical run, otherwise incremental will only read 7 days and anyhting prior to that may have already written dupes. you can add at the end of your dev process here, if you want the CI to remain quick(er)

Reply:
got you, thanks again for the explanation! adding the test 🤝